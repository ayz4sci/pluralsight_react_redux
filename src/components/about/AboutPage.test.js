import React from 'react';
import renderer from 'react-test-renderer';
import AboutPage from './AboutPage';

describe ('About Page Test', () => {
    it('renders correctly using snapshot', () => {
        const tree = renderer.create(<AboutPage />).toJSON();
        expect(tree).toMatchSnapshot();
    })
});