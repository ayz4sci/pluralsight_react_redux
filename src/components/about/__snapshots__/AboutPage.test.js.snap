// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Page Test renders correctly using snapshot 1`] = `
<div>
  <h1>
    About
  </h1>
  <p>
    This application uses React, Redux, React Router and a variety of other helpful libraries.
  </p>
</div>
`;
